optimum/habana/transformers/models/gptj/modeling_gptj.py

```diff
@@ -43,8 +43,6 @@ def __init__(self, config):
         self.q_proj = nn.Linear(self.embed_dim, self.embed_dim, bias=False)
         self.out_proj = nn.Linear(self.embed_dim, self.embed_dim, bias=False)
         self.rotary_dim = config.rotary_dim
-        pos_embd_dim = self.rotary_dim or self.embed_dim
-        self.embed_positions = create_sinusoidal_positions(max_positions, pos_embd_dim)
 
     def _split_heads(self, tensor, num_attention_heads, attn_head_size, rotary):
         """
```

Comment:
please add what the changes you have made for GPTJAttention refer [OPT](https://github.com/huggingface/optimum-habana/blob/main/optimum/habana/transformers/models/opt/modeling_opt.py#L12)

Reply:
added in the `forward` method in this class